src/components/dashboard/Dashboard.js

```diff
@@ -1,10 +1,62 @@
 import React from 'react'
 import './Dashboard.css';
+import FullCalendar from '@fullcalendar/react'
+import listPlugin from '@fullcalendar/list'
+import {ListWrapper} from './ListWrapper'
+
 
 const Dashboard = () => {
   return (
     <div className="Dashboard">
       <h1>THIS IS THE Dashboard</h1>
+      <ListWrapper>
+        <FullCalendar
+              plugins={[listPlugin]}
+              headerToolbar={{
+                left: '',
+                center: 'title',
+                right: ''
+              }}
+              views={{
+                upcoming: {
+                  type: 'list',
+                  duration: { days: 7 },
```

Comment:
This is great! I think seeing the upcoming week from *today* is going to be super helpful from a user perspective.